front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -67,7 +67,7 @@ const ReportBuilder = ({ report, reportLayout }: Props) => {
   const [imageUploading, setImageUploading] = useState(false);
   const [selectedLocale, setSelectedLocale] = useState<Locale>(platformLocale);
 
-  const [saved, setSaved] = useState(false);
+  const [saved, setSaved] = useState(true);
```

Comment:
@luucvanderzee It doesn't make any difference. And so, both in this branch and in master, when you open the report and click the "Back" button, it asks for confirmation (though it doesn't make sense as you just opened this page). Do you know how to fix it?